docs/how_to_contribute/math/how_to_add_prime_field.md

```diff
@@ -0,0 +1,409 @@
+# How to add Prime Field
+
+If you want to add new Prime Field, follow this guide.
+
+_Note_: We have our own development conventions. Please read the [conventions docs](/docs/how_to_contribute/conventions.md) before contributing.
+
+## 1. Add `BUILD.bazel` file
+
+Begin by creating a directory named `<prime_field_name>_prime` within `/tachyon/math/finite_field/`, and then add a `BUILD.bazel` file to it. After you have added parameters to your `BUILD.bazel`, Bazel will automatically generate the Prime Field code based on these parameters.
```

Comment:
isn't it that after the parameters are added and you build, Bazel auto generates it?
```suggestion
Begin by creating a directory named `<prime_field_name>_prime` in `/tachyon/math/finite_field/`. Add a `BUILD.bazel` into this directory. Note that once parameters are added to `BUILD.bazel`, Bazel will automatically generate the Prime Field code based on these parameters.
```

Reply:
^^wanted to ask this question oops "isn't it that after the parameters are added and you build, Bazel auto generates it?"